ansible/templates/usb/scripts/computenode.sh.j2

```diff
@@ -23,6 +25,14 @@ exec 4>>/dev/console
 # Time to wait for each VM to setup (in seconds)
 VM_DEPLOY_TIMEOUT=600
 
+BASE64="/opt/local/bin/base64"
+OPENSSL="/opt/local/bin/openssl"
+OPENSSL_HOME="/opt/local/etc/openssl"
+SVC_CERT_FILE="${OPENSSL_HOME}/dc-erigonesd.pem"
```

Comment:
We should not save the certificate into /opt/local on the compute node. Lets use /opt/custom/etc for this.